github3/repos/repo.py

```diff
@@ -2352,11 +2352,13 @@ def topics(self):
         json = self._json(self._get(url, headers=self.PREVIEW_HEADERS), 200)
         return self._instance_or_null(topics.Topics, json)
 
-    def tree(self, sha):
+    def tree(self, sha, recursive=False):
```

Comment:
@sigmavirus24 suggested [here](https://github.com/sigmavirus24/github3.py/issues/656#issuecomment-265740996) that the default value should be `True`, but this would have been a breaking change, so I preferred `False`.